hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsReadFooterMetrics.java

```diff
@@ -17,533 +17,348 @@
  */
 package org.apache.hadoop.fs.azurebfs.services;
 
```

Comment:
Can we have a parent abstract AbfsMetrics class and put common methods into it ?

Reply:
All the common methods are already placed in AbstractAbfsStatisticsSource abstract class. In AbfsReadFooterMetrics and AbfsBackoffMetrics classes, there is not much common method present.